### ISSUE TITLE
Syntax error in class ZCL_ABAPGIT_RES_REPOS due to formal parameter 'II_LOG' missing

### ISSUE THREAD

**Issue by GiantCrocodile** [abapGit/ADT_Backend#63]
Hello,

I tried to import this repository via abapGit into our NW 752 system. The goal is to be able to use the abapGit for Eclipse plugin. After pulling this repository I was able to activate it partially with abapGit and afterwards I had to activate some inactive objects manually. Now I face a syntax error in class ZCL_ABAPGIT_RES_REPOS because the formal parameter 'II_LOG' is unknown in line 74.

Can someone help me with this? Is this the right way to go or is there a newer solution to get the abapGit for Eclipse plugin to work? 

Best Regards,

**Comment by GiantCrocodile** [comment]
Furthermore I found out that:

- class `ZCL_ABAPGIT_RES_REPO_PUSH` complains about missing type `ZIF_ABAPGIT_SERVICES_GIT`
- class `ZCL_ABAPGIT_RES_REPO_PULL` misses the same parameter `II_LOG`

I'm not sure we fulfill the requirements: Does this work with NW 750 only or >= 750? What does the requirement `zabapgit version installed (report only won't work)` in the README mean? I have downloaded latest abapGit + executed it. abapGit does work fine on the system.

**Comment by absap** [comment]
Hello @GiantCrocodile 

did you run just the report?

"Report only won't work" means, that you have to install the sources additionally into the system.
Just create a new repo (https://docs.abapgit.org/guide-online-install.html) via zabapgit report and add as repo URL https://github.com/abapGit/abapGit/.
Then also the sources are installed into the system.

**Comment by GiantCrocodile** [comment]
Hi @absap! Thank you for your quick response. You were right about what I did wrong. I didn't clone the abapGit sources and thus some technical objects were missing. After I did pull the repository you did mention, it did work. I'm able to use the abapGit plugin in Eclipse successfully. I tested this by retrieving the list of known repositories in the SAP system.

I will create a pull request to clarify the readme.
